src/modules/jobs/jobs.service.ts

```diff
@@ -69,6 +69,27 @@ export class JobsService {
       data: job,
     };
   }
+
+  async update(id: string, updateJobDto: Partial<JobDto>) {
+    const job = await this.jobRepository.findOne({ where: { id } });
+    if (!job) {
+      throw new NotFoundException({
+        status_code: 404,
+        status: 'Not found Exception',
+        message: 'Job not found',
```

Comment:
Better to use constants where possible. This makes code cleaner and maintainable